### PR TITLE
chore: update Talos to 1.7.2 in the tests

### PR DESCRIPTION
**Pull request by Unix4ever**
Bump machinery to 1.7.2.
Enable partial machine config tests.

### CODE REVIEW

**Comment by smira** [comment]
@Unix4ever should we update the defaults?

**Comment by Unix4ever** [comment]
> @Unix4ever should we update the defaults?

Yep, sure

**Review by smira — APPROVED** [review]
gentle reminder about bumping default Talos/Kubernetes version

**Review by utkuozdemir — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -20,9 +20,9 @@ REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 PROTOBUF_GRPC_GATEWAY_TS_VERSION ?= 1.2.1
 TESTPKGS ?= ./...
 NODE_BUILD_ARGS ?=
-PROTOBUF_GO_VERSION ?= 1.34.1
+PROTOBUF_GO_VERSION ?= 1.33.0
 GRPC_GO_VERSION ?= 1.3.0
-GRPC_GATEWAY_VERSION ?= 2.20.0
+GRPC_GATEWAY_VERSION ?= 2.19.1
```
Comment (utkuozdemir) [inline Makefile:25]: These going down is expected I guess?

**Review by utkuozdemir — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Unix4ever) [reply Makefile:25]: Yeah. We might need to look into gRPC proxy to check if it supports that properly.
With these versions enabled all Omni API stops working.

**Pull request merged**